portal/templates/admin.html

```diff
@@ -64,4 +66,26 @@ <h3 class="tnth-headline">User Administration</h3>
     $('#adminTable').bootstrapTable('filterBy', {roles: "patient"});
 });
 **/
+
+function deleteUser(event, userId) {
+    if (event) event.stopPropagation();
+    if (userId) {
+       var c = confirm("Are you sure you want to delete this user?");
+       if (c) {
+         $.ajax ({
+              type: "GET",
```

Comment:
We are attempting to stick with a RESTful API layout - I think a better endpoint would be to use the DELETE verb (in place of GET) on /api/user/{user_id}